resources/views/components/button/index.blade.php

```diff
@@ -0,0 +1,15 @@
+@props(['disableWhenLoading' => true])
```

Comment:
Should we keep this or reverse it?

Reply:
I mean; it's now opt-out, maybe opt-in is better? Opt-out makes sure nobody forgets it but maybe it makes more sense to flip this? Just as a thing to think over instead of sticking with what it was.